apps/glusterfs/app.go

```diff
@@ -256,6 +289,21 @@ func (a *App) setAdvSettings() {
 	}
 }
 
+func (a *App) setBlockSettings() {
```

Comment:
If values is not correct then this will fall back to default. I will add that correctness.